docs/app/Examples/collections/Grid/Variations/GridTextAlignmentJustifiedExample.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Grid, Menu } from 'stardust'
+
+// TODO: Update <Menu> usage after v1 API
+
+const { Column, Row } = Grid
+
+const GridTextAlignmentJustifiedExample = () => (
+  <Grid textAlign='center'>
```

Comment:
It seems the left aligned column inside of the center aligned grid is also not working.  This might add to the confusion.

Since there is already an example showing alignment on the grid itself, what if we remove `textAlign='center'` from the grid here.  Then, change the column `textAlign='left'` below to `textAlign='center'` so it shows column override alignment.  Finally, leave the row `textAlign='justified'` as is.  Then this example will show left, center, and justified alignments.

I like `GridTextAlignmentMixedExample.js` as well.